README.md

```diff
@@ -69,25 +52,59 @@ For example on *nix:
     Run the script named fits.sh
         ./fits.sh
 
-Here are a couple examples of running FITS to get you started. These are relatively simple examples assuming Windows - more complex examples can be found in the on-line user manual. 
+Here are a couple examples of running FITS to get you started. These are relatively simple examples assuming Windows -
+more complex examples can be found in the on-line user manual. 
 
     Run FITS against its release text file printing the FITS output to the terminal: fits.bat -i version.properties (or, on Linux, ./fits.sh -i version.properties)
     Run FITS against its release text file saving the FITS output to a file: fits.bat -i version.properties -o myoutput.txt
     Output the technical metadata only (in the TextMD format) for the file to the terminal: fits.bat -x -i version.properties
     Output the FITS output plus technical metadata (in the TextMD format) for the text file to the terminal: fits.bat -xc -i version.properties
 
+FITS through Docker
+-------------------
+In the root of the FITS distribution zip is a Dockerfile that can be used to build a FITS Docker image. To build the
+image, execute the following from within the FITS root directory:
+
+```shell
+docker build -f Dockerfile -t fits:latest -t fits:1.6.0 .
+```
+
+After building the image, you can use it directly to analyze files. The following are some examples. Note these
+examples mount the current working directly within the Docker container, which means that the only files that are
+accessible within the container are files that are relative the current working directory. Additionally, these commands
+**do not** need to be run within the FITS root and can be run anywhere on the system.
+
+```shell
+# Run FITS on a file
+docker run --rm -v `pwd`:/work:z fits -i file.txt
```

Comment:
Is the `z` flag actually necessary?

Reply:
Depends on your system. It is necessary if you're using Podman on Fedora, and doesn't hurt :)

Reply:
I am happy to leave it in, but would be interested to understand the decision-making process for it's inclusion: https://docs.docker.com/storage/bind-mounts/#configure-the-selinux-label. It seems to only apply to [RedHat SELinux](https://www.redhat.com/en/topics/linux/what-is-selinux).